src/LoRa.cpp

```diff
@@ -607,9 +607,21 @@ void LoRaClass::setOCP(uint8_t mA)
   writeRegister(REG_OCP, 0x20 | (0x1F & ocpTrim));
 }
 
+void LoRaClass::LoRandomSeed() {
```

Comment:
What do you think about calling this API `LoRa.beginRandom()` (so it's like `LoRa.beginPacket()`) - then there could also be a `LoRa.endRandom()` that could put the module in idle()?

Reply:
That's an excellent idea.